### PR TITLE
BETA: Register mostafizur+rahman.is-a.dev

### PR DESCRIPTION
**Pull request by gigwanoobstudios**
Added `mostafizur+rahman.is-a.dev` using the [dashboard](https://manage.is-a.dev).

### CODE REVIEW

**Comment by gigwanoobstudios** [comment]
Had to use "+" for email and my old domain without"+" is not showing😑
Is there any way to fix it

**Comment by WilliamDavidHarrison** [comment]
> Had to use "+" for email and my old domain without"+" is not showing😑 Is there any way to fix it

That domain seems to have been deleted in the recent domain cleanup #9887

File: domains/mostafizur+rahman.json
Comment (DEV-DIBSTER) [inline domains/mostafizur+rahman.json:1]: Invalid domain name.

**Review by DEV-DIBSTER — DISMISSED** [review]

File: domains/mostafizur+rahman.json
Comment (gigwanoobstudios) [reply domains/mostafizur+rahman.json:1]: Changed to mostafizur-rahman.is-a.dev

**Review by wdhdev — APPROVED** [review]

**Pull request merged**